File: Tutorial-1-SampleCode/Tutorial-1-SampleCode/args/CmdArgs.c
#include <stdio.h>

int main( int argc, char **argv ) {
	int i;
	printf( "Number of command-line arguments: " );
	printf( "%d\n", argc );

	for ( i = 0; i < argc; i++ ) {
		printf( "Argument #%d: ", i );
		printf( "%s\n", argv[ i ] );
	}

	return 0;
}

File: Tutorial-1-SampleCode/Tutorial-1-SampleCode/args/CmdArgs.java
public class CmdArgs {
	public static void main( String[] args ) {
		System.out.print( "Number of command-line arguments: " );
		System.out.println( args.length );

		for ( int i = 0; i < args.length; i++ ) {
			System.out.print( "Argument #" + i + ": " );
			System.out.println( args[ i ] );
		}
	}
}
